[2020-06-03 08:15:22,062] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-03 08:15:22,076] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-03 08:15:22,076] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:15:22,076] {taskinstance.py:867} INFO - Starting attempt 6 of 7
[2020-06-03 08:15:22,076] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:15:22,096] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): covid_data> on 2020-05-30T00:00:00+00:00
[2020-06-03 08:15:22,102] {standard_task_runner.py:53} INFO - Started process 30073 to run task
[2020-06-03 08:15:22,169] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_dag.covid_data 2020-05-30T00:00:00+00:00 [running]> nineleaps
[2020-06-03 08:15:22,618] {taskinstance.py:1128} ERROR - string indices must be integers
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/nineleaps/airflow/dags/covid_data.py", line 69, in get_covid_data
    csv_writer.writerow([str(context['execution_date']).split('T')[0], i['State UT'], i[str(context['execution_date']).split('T')[0]]])
TypeError: string indices must be integers
[2020-06-03 08:15:22,621] {taskinstance.py:1151} INFO - Marking task as UP_FOR_RETRY
[2020-06-03 08:15:32,051] {logging_mixin.py:112} INFO - [2020-06-03 08:15:32,051] {local_task_job.py:103} INFO - Task exited with return code 1
